src/tree/StaticWebAppTreeItem.ts

```diff
@@ -84,11 +84,16 @@ export class StaticWebAppTreeItem extends AzureParentTreeItem implements IAzureR
     public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtTreeItem[]> {
         const requestOptions: requestUtils.Request = await requestUtils.getDefaultAzureRequest(`${this.id}/builds?api-version=2019-12-01-preview`, this.root);
         const envs: StaticEnvironment[] = (<{ value: StaticEnvironment[] }>JSON.parse(await requestUtils.sendRequest(requestOptions))).value;
+        const remote: string | undefined = await tryGetRemote();
+        const branch: string | undefined = remote ? await tryGetBranch() : undefined;
```

Comment:
How often does `tryGetRemote` get called related to this PR? Like - if I have a bunch of static apps or environments - does it get called a lot? If so, we might want to find a more efficient way of caching this information since it's tied to the workspace and is unlikely to change

Reply:
Right now it gets called:
- When an `EnvironmentTreeItem` is refreshed
- When a `StaticWebAppTreeItem` is expanded (only called once per expanding, not per environment)
- When creating a Static Web App (during the wizard)

I see the concern, but I don't think it'll really impact users too much.  It's a pretty fast GET request, the bottleneck would probably be making the request to Azure for the children.  

I'm also thinking we can remove the check in `EnvironmentTreeItem` (though we should probably support multi-root at some point)